packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/core/GestureHandler.kt

```diff
@@ -886,6 +887,9 @@ open class GestureHandler {
       if (config.hasKey(KEY_DISPATCHES_ANIMATED_EVENTS)) {
         handler.dispatchesAnimatedEvents = config.getBoolean(KEY_DISPATCHES_ANIMATED_EVENTS)
       }
+      if (config.hasKey(KEY_SHOULD_USE_REANIMATED)) {
+        handler.dispatchesReanimatedEvents = config.getBoolean(KEY_SHOULD_USE_REANIMATED)
+      }
```

Comment:
I think `dispatchesReanimatedEvents` and `dispatchesAnimatedEvents` also isn't reset in `resetConfig`.